CHANGELOG.rst

```diff
@@ -1,13 +1,10 @@
 3.0.4.dev
 =========
 
-*
-
-*
-
-*
+**Bug Fixes**
 
-*
+* Minor fix to the documentation.
+  Thanks `@budulianin`_ for the PR.
```

Comment:
I don't think a changelog entry is needed for this :wink:

If you want to keep it, you should not remove the empty lines (which are here to avoid merge conflicts), and would need to add a link reference for your github username.